apache2/apache2_config.c

```diff
@@ -1529,6 +1529,10 @@ static const char *cmd_component_signature(cmd_parms *cmd, void *_dcfg,
     /* ENH Enforce "Name/VersionX.Y.Z (comment)" format. */
     *(char **)apr_array_push(dcfg->component_signatures) = (char *)p1;
 
+#ifdef WAF_JSON_LOGGING_ENABLE
+    msc_waf_signature = (char *)p1;
```

Comment:
can there be multiple `SecComponentSignature`? What happens to your solution in that case?

Reply:
since it's very specific to our config, i checked there is only one SecComponentSignature for one ruleset, otherwise it will be the last one which overwriting the previous